core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -614,7 +584,6 @@ oppia.directive('conversationSkin', [
             }
             $scope.adjustPageHeight(false, null);
             $window.scrollTo(0, 0);
-            FocusManagerService.setFocusIfOnDesktop(_nextFocusLabel);
```

Comment:
What effect does removing this have?

Reply:
I'm not sure of this, should I try removing this? It was present in the existing code as so I used it.

Reply:
OK, so here's how I think this works. When generating the interaction HTML, a focus label in inserted into it. This helps us autofocus on the right thing as soon as the interaction is shown to the learner.

So -- we do need to have a call to setFocusIfOnDesktop() at the right time in ConversationSkinDirective. So this probably needs to go somewhere -- if you removed it, I hope it's put back somewhere in the right place.

Reply:
Done, added setFocus() function to after a card is actually added.